assets/src/apps/authoring/components/PropertyEditor/schemas/screen.ts

```diff
@@ -116,16 +116,32 @@ export const screenUiSchema = {
   },
 };
 
-export const getScreenData = (data: any) => {
+export const transformScreenModeltoSchema = (data: any) => {
```

Comment:
in this method we need to do the conversion from the old properties
so the schema palette should only have the backgroundColor etc type properties as it does, but need to test
```
if (!data.palette.useHtmlProps) {
schema.palette.backgroundColor = `rgba(${
                    data.palette?.fillColor ||
                    data.palette?.fillColor === 0
                        ? chroma(data.palette?.fillColor).rgb().join(',')
                        : '255, 255, 255';
}
```
etc... this is similar to the code found here:
https://github.com/Simon-Initiative/oli-torus/blob/master/assets/src/apps/delivery/layouts/deck/DeckLayoutView.tsx#L398